pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -253,38 +288,52 @@ func (r *reconciler) BatchReconcile(rules []*CompletedRule) error {
 		return err
 	}
 	for _, rule := range rulesToInstall {
+		ruleTable := r.getOFRuleTable(rule)
 		klog.V(2).Infof("Adding rule %s of NetworkPolicy %s/%s to be reconciled in batch", rule.ID, rule.PolicyNamespace, rule.PolicyName)
-		ofPriority, _ := r.getOFPriority(rule)
+		ofPriority, _ := r.getOFPriority(rule, ruleTable)
 		priorities = append(priorities, ofPriority)
+		if ofPriority != nil {
+			prioritiesByTable[ruleTable] = append(prioritiesByTable[ruleTable], ofPriority)
+		}
 	}
 	ofRuleInstallErr := r.batchAdd(rulesToInstall, priorities)
 	if ofRuleInstallErr != nil {
-		for _, ofPriority := range priorities {
-			if ofPriority != nil {
-				r.priorityAssigner.Release(*ofPriority)
+		for tableID, ofPriorities := range prioritiesByTable {
+			for _, ofPriority := range ofPriorities {
+				r.priorityAssigners[tableID].Release(*ofPriority)
 			}
 		}
 	}
 	return ofRuleInstallErr
 }
 
+// registerOFPriorities make the Priorities corresponding to each CompletedRule
+// known to the priorityAssigner for each OpenFlow table.
```

Comment:
The sentence seems hard to understand. Consider to revise it.

Reply:
Sure, will revise